dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityChangeLog.java

```diff
@@ -27,20 +27,80 @@
  */
 package org.hisp.dhis.tracker.export.trackedentity;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Date;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hisp.dhis.changelog.ChangeLogType;
+import org.hisp.dhis.program.ProgramTrackedEntityAttribute;
 import org.hisp.dhis.program.UserInfoSnapshot;
+import org.hisp.dhis.trackedentity.TrackedEntity;
+import org.hisp.dhis.trackedentity.TrackedEntityAttribute;
 
-public record TrackedEntityChangeLog(
-    @JsonProperty UserInfoSnapshot createdBy,
-    @JsonProperty Date createdAt,
-    @JsonProperty String type,
-    @JsonProperty Change change) {
+@NoArgsConstructor
```

Comment:
Are

@NoArgsConstructor
@Setter

needed for hibernate?

Reply:
yes